src/unity/python/turicreate/toolkits/activity_classifier/_activity_classifier.py

```diff
@@ -28,6 +28,7 @@
 from turicreate.toolkits._model import PythonProxy as _PythonProxy
 
 from .util import random_split_by_session as _random_split_by_session
+from .util import _MIN_NUM_SESSIONS_FOR_SPLIT
```

Comment:
Actually - you could just move this param to this file. It's not needed in the util file anymore.

Reply:
I'm not a python expert, but AFAICT moving this constant from util to _activity_classifier means that either
* test_activity_classifier must copy the constant, so that it is defined in two places, or
* toolkits/activity_classifier/__init__.py must export it as a fourth symbol, after `['create', 'ActivityClassifier', 'util']`.

For now, I just went with leaving it in the util.py module